applications/conf/routes

```diff
@@ -26,7 +26,7 @@ GET            /crosswords/search
 GET            /crosswords/lookup                                                                                                controllers.CrosswordSearchController.lookup()
 
 # Email paths
-GET        /email/form/$emailType<article|footer|plain|plaindark|plaintone>/:listId       controllers.EmailSignupController.renderForm(emailType: String, listId: Int)
+GET        /email/form/$emailType<article|footer|plain|plaindark|plaintone>/:listId       controllers.EmailSignupController.renderForm(emailType: String, listId: String)
```

Comment:
Looking at the code, I think it might be cleaner to have a second route (with lower precedence) that is a `listName: String`. Then call a different controller method, that reference calls the one for `listId: Int` once it has done the `fromIdentityName` dance.  Can probably avoid the `Try` stuff then.

Reply:
Have done this with some regex to route the requests to the correct method